bitnami/jupyterhub/Chart.yaml

```
@@ -37,4 +37,4 @@ maintainers:
 name: jupyterhub
 sources:
 - https://github.com/bitnami/charts/tree/main/bitnami/jupyterhub
-version: 5.2.9
+version: 5.2.10
```

As I understand `hub.services` was not supported before and with these changes you are adding this feature. According to [semver](https://semver.org/) a new minor version should be released:
```suggestion
version: 5.3.0
```